app/javascript/dashboard/helper/flag.js

```diff
@@ -0,0 +1,19 @@
+const FLAG_OFFSET = 127397;
+
+/**
+ * Gets emoji flag for given locale.
+ *
+ * @param {string} countryCode locale code
+ * @return {string} emoji flag
+ *
+ * @example
+ * getCountryFlag('cz') // '🇨🇿'
+ */
+export const getCountryFlag = countryCode => {
```

Comment:
<img width="551" alt="Screenshot 2022-07-28 at 8 06 46 PM" src="https://user-images.githubusercontent.com/2246121/181556562-1fba7936-4b75-44aa-b1e4-354795b09c15.png">


Let us revert this change. The country-code-emoji does the exact same thing. I don't there is a significant advantage of us managing the code.